specification/containerservice/resource-manager/Microsoft.ContainerService/preview/2022-06-02-preview/fleets.json

```diff
@@ -686,6 +728,37 @@
           "description": "The provisioning state of the last accepted operation."
         }
       }
+    },
+    "CredentialResults": {
+      "type": "object",
+      "properties": {
+        "kubeconfigs": {
+          "type": "array",
+          "readOnly": true,
+          "items": {
+            "$ref": "#/definitions/CredentialResult"
+          },
+          "description": "struct hold multiple Kubernetes client configuration file."
```

Comment:
Minor: Don't use word `struct` here as in many contexts (Python, C#) it either doesn't exist, or it is not being used. For example in C# this is a class not a struct.

I would instead say: "The list of Kubernetes client configuration files" or possibly "The list of Kubernetes configs for authenticating with the cluster"